packages/@aws-cdk/aws-codepipeline-actions/lib/cloudformation/pipeline-actions.ts

```diff
@@ -513,6 +513,487 @@ export class CloudFormationDeleteStackAction extends CloudFormationDeployAction
   }
 }
 
+/**
+ * Determines how IAM roles are created and managed
```

Comment:
Comments should end with a period:

```suggestion
 * Determines how IAM roles are created and managed.
```

Some comment everywhere below.